apps/dashboard_app/helpers/protocol_stats.py

```diff
@@ -253,5 +253,7 @@ def get_utilization_stats(
         },
     )
     utilization_columns = [x for x in data.columns if "utilization" in x]
-    data[utilization_columns] = data[utilization_columns].map(lambda x: round(x, 4))
+    # NOTE: This was made so test can pass, something to look into
```

Comment:
DO not change code not related to your task, please revert it

Reply:
am getting an error `AttributeError: 'DataFrame' object has no attribute 'map'. Did you mean: 'max'?`
this error is independent of the input supplied by the test.

`apply()` does exactly what is expected i.e apply the rounding action on every element in the DataFrame